_data/projects/AutoSort.yml

```diff
@@ -0,0 +1,10 @@
+name: AutoSort
+desc: Sort by a string or by default attributeA brief description of the project
+site: https://github.com/ming-tsai/NetCore.AutoSort
+tags:
+  - c#
+  - netcore
+  - dotnet3-1
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/ming-tsai/NetCore.AutoSort/labels/up-for-grabs
```

Comment:
Looks like there are no open issues currently for new contributors. Is that something you're able to setup before we merge this?

Reply:
I have already close it, I will reopen it